salt/modules/mac_brew_pkg.py

```diff
@@ -353,6 +370,10 @@ def remove(name=None, pkgs=None, **kwargs):
         A list of packages to delete. Must be passed as a python list. The
         ``name`` parameter will be ignored if this option is passed.
 
+    options
```

Comment:
Please indicate that this new argument is expected to be a list of strings

Reply:
Done!

Reply:
Hi @Akm0d, I already did the requested changes.

Please, could you check changes are OK?

Thank you!